app/kubernetes/views/applications/applicationsController.js

```diff
@@ -102,8 +109,20 @@ class KubernetesApplicationsController {
 
   async getApplicationsAsync() {
     try {
-      const applications = await this.KubernetesApplicationService.get();
-      this.applications = applications;
+      const [applications, configurations] = await Promise.all([this.KubernetesApplicationService.get(), this.KubernetesConfigurationService.get()]);
+      const configuredApplications = KubernetesConfigurationHelper.getApplicationConfigurations(applications, configurations);
+      const helmApplications = KubernetesApplicationHelper.getHelmApplications(configuredApplications);
+
+      // filter out multi-chart helm managed applications
+      const helmAppNames = helmApplications.map((hma) => hma.Name);
+      const nonHelmApps = configuredApplications.filter(
+        (app) =>
+          !app.Pods.flatMap((pod) => pod.Labels) // flatten pod labels
+            .filter((label) => label) // filter out empty labels
+            .some((label) => helmAppNames.includes(label['app.kubernetes.io/instance'])) // check if label key is in helmAppNames
```

Comment:
use constant label

Reply:
Will address endpoint provider issue in another ticket (base epic ticket) - as this PR only deals with helm list/uninstall.